syntax/ps1.vim

```diff
@@ -82,7 +82,7 @@ syn match ps1Type /\[[a-z_][a-z0-9_.,\[\]]\+\]/
 " Variable references
 syn match ps1ScopeModifier /\(global:\|local:\|private:\|script:\)/ contained
 syn match ps1Variable /\$\w\+\(:\w\+\)\?/ contains=ps1ScopeModifier
-syn match ps1Variable /\${\w\+\(:\w\+\)\?}/ contains=ps1ScopeModifier
+syn match ps1Variable /\${[^`}]\+}/ contains=ps1ScopeModifier
```

Comment:
That means any character but `}`, which is too permissive. Probably best to use `[\w()]\+` or, better, consult the PowerShell source to see exactly which characters are permitted in a variable name.

Reply:
It's any character but `}` and backtick, which I got from the docs. I will check the tokenizer in the PowerShell source.